plugins/pencilblue/controllers/error/404.js

```diff
@@ -31,45 +31,42 @@ module.exports = function NotFoundControllerModule(pb) {
      * @extends BaseController
      */
     function NotFoundController(){}
-    util.inherits(NotFoundController, pb.BaseController);
-
+    util.inherits(NotFoundController, pb.ErrorViewController);
```

Comment:
Updated the controller to leverage the default error controller.  This provides backward compatibility as well as reduces LOC.